apollo-core/src/main/java/com/ctrip/framework/foundation/internals/NetworkInterfaceManager.java

```diff
@@ -119,15 +119,10 @@ private void load() {
 
     try {
       Enumeration<NetworkInterface> interfaces = NetworkInterface.getNetworkInterfaces();
-      List<NetworkInterface> nis = interfaces == null ? Collections.<NetworkInterface>emptyList()
-          : Collections.list(interfaces);
+      List<NetworkInterface> nis = new ArrayList<>(interfaces == null ? Collections.emptyList()
+              : Collections.list(interfaces));
```

Comment:
Is `new ArrayList<>` necessary?
```suggestion
      List<NetworkInterface> nis = interfaces == null ? Collections.emptyList()
              : Collections.list(interfaces);
```